packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -118,7 +118,10 @@ class HorizontalOverflow extends Foundation<
             >
                 <div
                     style={{
-                        height: `${this.state.itemsHeight}px`,
+                        height:
+                            this.state.itemsHeight !== null
+                                ? `${this.state.itemsHeight}px`
+                                : "unset",
```

Comment:
I don't believe `unset` is supported whatsoever in IE. https://developer.mozilla.org/en-US/docs/Web/CSS/unset

Reply:
We should validate it works correctly in IE, but I expect it does - the property should be ignored if the value is unrecognized, which *should* have the same effect as "unset". Another option would be to assign `height` to `undefined` and let React filter the property out.

Reply:
If React filters out undefined then that is likely the better alternative (that have the possibility of IE give a warning/undefined behavior on a property)

Reply:
Updated to auto per @janechu's comment, resolving this conversation to only have one thread open on the issue